backends/arm/arm_backend.py

```diff
@@ -38,6 +38,118 @@
     logger.setLevel(logging.INFO)
 
 
+class ArmCompileSpecBuilder:
+    def __init__(self):
+        self.compile_spec: List[CompileSpec] = []
+        self.compiler_flags = []
+        self.output_format = None
+        self.path_for_intermediates = None
+        self.permute_nhwc = False
+
+    def ethosu_compile_spec(
```

Comment:
why not just do this in `__init__` or you can create a `@classmethod` if you want to write another constructor.